### PR TITLE
Change GSDML file to Profinet version 2.4

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml [hunk 1]
```diff
@@ -176,11 +176,22 @@
...
+               <!--Error code 0x20  Error decode 0x82  Error code 1 0x4E-->
+               <ErrorCode2Value>
+                  <Name TextId="IDT_CUSTOM_LOGBOOK_1"/>
+               </ErrorCode2Value>
+            </LogBookEntryItem>
+         </LogBookEntryList>
          <ExternalTextList>
             <PrimaryLanguage>
-               <Text TextId="IDT_INFO_Device" Value="PNO GSDML V2.35 Examples"/>
+               <Text TextId="IDT_INFO_Device" Value="PNO GSDML V2.4 Examples"/>
```
Comment (olbjo) [inline sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml:189]: PNIO

File: sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml [hunk 2]
```diff
@@ -176,11 +176,22 @@
                </VirtualSubmoduleList>
             </ModuleItem>
          </ModuleList>
+         <LogBookEntryList>
+            <LogBookEntryItem Status="2130510">
```
Comment (olbjo) [inline sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml:180]: I do not get where this comes from. Maybe a comment should be added

**Review by olbjo — CHANGES_REQUESTED** [review]
Great!
Minor spelling error detected.
I do not get the logbook.

File: sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml (hunk 2 above)
Comment (pyhys) [reply sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml:180]: Done

File: sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml (hunk 1 above)
Comment (pyhys) [reply sample_app/GSDML-V2.4-rtlabs-IODevice-20201211.xml]: Done

**Review by olbjo — APPROVED** [review]

**Pull request merged**